Tests/ScipioKitTests/DescriptionPackageTests.swift

```diff
@@ -76,17 +76,13 @@ final class DescriptionPackageTests: XCTestCase {
         let graph = try package.resolveBuildProductDependencyGraph()
             .map { $0.target.name }
 
-        let myPluginNode = try XCTUnwrap(graph.rootNodes.first)
-        XCTAssertEqual(myPluginNode.value, "MyPlugin")
-
-        let executableTargetNode = try XCTUnwrap(myPluginNode.children.first)
-        XCTAssertEqual(executableTargetNode.value, "ExecutableTarget")
```

Comment:
Is is actually oaky to remove these assertions?

ref:
- #200

Reply:
Yes, since `PackageResolver` filters targets to only `regular` and `binary` types, I removed this test assertion.


https://github.com/Ryu0118/Scipio/blob/5f4d76b5e2a011f8a04948c06624fd6ae28af701/Sources/ScipioKit/Resolver/PackageResolver.swift#L74
https://github.com/Ryu0118/Scipio/blob/5f4d76b5e2a011f8a04948c06624fd6ae28af701/Sources/ScipioKit/Resolver/PackageResolver.swift#L206